spec/spec_helper.rb

```diff
@@ -11,3 +11,11 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 end
+
+shared_context "user in an active call" do
```

Comment:
It make sense why you wanted to pull this out to make it more reusable, but I think trying to make tests dry obscure how a test is being set up and make them harder to maintain. What are you thoughts on pulling this back in to the main body of the test?

This article talks about avoiding `let` blocks, but the same ideas apply: [Let's Not](https://thoughtbot.com/blog/lets-not)
This one, too, advocating for WET tests: [The case for WET Tests](https://thoughtbot.com/blog/the-case-for-wet-tests)